docs/UserGuide.md

```diff
@@ -120,7 +149,11 @@ Finds a tag by its keyword (can be tag name and/or label).
 Format: `find KEYWORD`
         
 Examples:
-* `find notes`
+* `find file1` (finds a tag with tag name `file1`)
+* `find label2020` (finds a tag with label name `label2020`)    
+* `find tagname1 label1` (finds a tag name `tagname1` and label name `label1`)
```

Comment:
I think Find Command does not distinguish between tag and label. So this will find any tag with tag name or label that contain "tagname1" or "label1"?

Reply:
oh right